lib/commands/-command.js

```diff
@@ -6,7 +6,7 @@ module.exports = Command.extend({
   init() {
     process.env.EMBER_CLI_ELECTRON = true;
 
-    if (this._super && this._super.init) {
+    if (this._super && this._super.init && this._super.init !== this.init) {
```

Comment:
Can't this whole block just be `this._super(...arguments)` (or `this._super.apply(this, arguments)` if we care about older Node compatibility)?